frontend/src/components/QueryBuilderHandler/utils/index.ts

```diff
@@ -0,0 +1,149 @@
+import Tokenizr from 'tokenizr';
+import { Set } from 'immutable';
+
+export const parseAdvancedQueryString = (options: string, columns: Set<string>): any => {
```

Comment:
Curious ... can this build anything besides filters? Also will be used outside of Filter components? If not, move it out of the general utils folder for the query builder.

Reply:
Also, I'm sure you can get rid of all these `any` types.

Reply:
Moved `parseAdvancedQueryString` function to appropriate utils folder and updated `any` types